shared/src/packages/users/libs/validation-schemas/enums/regexp.enum.ts

```diff
@@ -0,0 +1,9 @@
+const Regexp = {
```

Comment:
Let's rename this one. UserValidationRule or something

Reply:
Agree

Reply:
renamed to UserValidationRules